lib/Db/RowCellMapperSuper.php

```diff
@@ -72,6 +72,22 @@ public function deleteAllForRow(int $rowId): void {
 		$qb->executeStatement();
 	}
 
+	/**
+	 * @throws Exception
+	 */
+	public function deleteAllForRowAndColumn(int $rowId, int $columnId): void {
+		$qb = $this->db->getQueryBuilder();
+		$qb->delete($this->tableName)
+			->where(
+				$qb->expr()->eq('row_id', $qb->createNamedParameter($rowId, IQueryBuilder::PARAM_INT))
+			)
+			->andWhere(
+				$qb->expr()->eq('column_id', $qb->createNamedParameter($columnId, IQueryBuilder::PARAM_INT))
+			);
```

Comment:
The column order may matter when using database indices, however in the case of this simple query it seems to work fine and still use the index that is `(column_id, row_id)`



```
EXPLAIN select * from oc_tables_row_cells_text where row_id = 1 and column_id = 2;

1	SIMPLE	oc_tables_row_cells_text	ref	IDX_3AC41548BE8E8ED583A269F2	IDX_3AC41548BE8E8ED583A269F2	8	const,const	1	
```

Reply:
Oh, good point. Maybe still better to turn it around? To be safe and be a good guide.